internal/view/container.go

```diff
@@ -68,9 +70,10 @@ func (c *Container) bindKeys(aa ui.KeyActions) {
 	}
 
 	aa.Add(ui.KeyActions{
-		ui.KeyF:      ui.NewKeyAction("Show PortForward", c.showPFCmd, true),
-		ui.KeyShiftF: ui.NewKeyAction("PortForward", c.portFwdCmd, true),
-		ui.KeyShiftT: ui.NewKeyAction("Sort Restart", c.GetTable().SortColCmd("RESTARTS", false), false),
+		ui.KeyF:        ui.NewKeyAction("Show PortForward", c.showPFCmd, true),
+		ui.KeyShiftF:   ui.NewKeyAction("PortForward", c.portFwdCmd, true),
+		ui.KeyShiftT:   ui.NewKeyAction("Sort Restart", c.GetTable().SortColCmd("RESTARTS", false), false),
+		tcell.KeyCtrlY: ui.NewKeyAction("Copy container image", c.copyImageToClipboard, true),
```

Comment:
We are already in the context of container, so perhaps `Copy image` makes better sense. Not to mention keeping action name short.

Reply:
resolved @derailed